drivers/tty/serdev/pibridge_trace.h

```diff
@@ -131,17 +204,36 @@ DECLARE_EVENT_CLASS(pibridge_receive_crc_class,
 	)
 );
 
+/*
+ * pibridge_receive_io_crc
+ *
+ * Info: the CRC that is part of the received IO packet.
+ * Time: after the CRC of an IO packet has been received.
+ */
 DEFINE_EVENT(pibridge_receive_crc_class, pibridge_receive_io_crc,
 	TP_PROTO(u8 crc, u8 exp_crc),
 	TP_ARGS(crc, exp_crc)
 );
 
+/*
+ * pibridge_receive_gate_crc
+ *
+ * Info: the CRC that is part of the received gateway packet.
+ * Time: after the CRC of a gateway packet has been received.
+ */
 DEFINE_EVENT(pibridge_receive_crc_class, pibridge_receive_gate_crc,
 	TP_PROTO(u8 crc, u8 exp_crc),
 	TP_ARGS(crc, exp_crc)
 );
 
-/* packet data */
+/*
+ * pibridge_buffer_data_class
+ *
+ * Print a data buffer and the buffer length.
+ *
+ * datalen: number of data bytes.
+ * data: data bytes.
```

Comment:
s/data:/buffer:/
s/datalen:/len:/

Reverse order of the above two lines to fit TP_PROTO() macro below.

Reply:
Never mind, I didn't realize that you're specifying the data emitted by the tracepoint, not its arguments.